cypress/integration/smoke/balances.spec.js

```diff
@@ -0,0 +1,196 @@
+
+      // Link to show details
+      cy.get(balanceRowTestId).eq(WETH_ROW_NUMBER).get('a[aria-label="Show details on Etherscan"]').should('exist')
+
+      // Balance should end with WETH
+      cy.get(balanceRowTestId).eq(WETH_ROW_NUMBER).find('td').eq(TOKEN_AMOUNT_COLUMN).contains('WETH')
+    })
+
+    it('should have USD Coin', () => {
+      const USD_ROW_NUMBER = 3
+
+      cy.get(balanceRowTestId).contains('USD Coin')
+
+      // First row should have a image with alt text "USD Coin"
+      cy.get(balanceRowTestId).eq(USD_ROW_NUMBER).get('img[alt="USD Coin"]').should('exist')
+
+      // Link to show details
+      cy.get(balanceRowTestId).eq(USD_ROW_NUMBER).get('a[aria-label="Show details on Etherscan"]').should('exist')
+
+      // Balance should end with USDC
+      cy.get(balanceRowTestId).eq(USD_ROW_NUMBER).find('td').eq(TOKEN_AMOUNT_COLUMN).contains('USDC')
+    })
+  })
+
+  describe('values should be formatted as per locale', () => {
+    it('should have Token balance formated as per tokenRegex', () => {
+      // Balance should comply with the tokenRegex
+      cy.get(assetsTableContainer)
+        .find(balanceRowTestId)
+        .eq(1)
+        .find('td')
+        .eq(TOKEN_AMOUNT_COLUMN)
+        .should(($div) => {
+          const text = $div.text()
+          expect(text).to.match(tokenRegex)
+        })
+    })
+
+    it('should have Fiat balance formated as per fiatRegex', () => {
+      // Balance should comply with the fiatRegex
+      cy.get(assetsTableContainer)
+        .find(balanceRowTestId)
+        .eq(1)
+        .find('td')
+        .eq(FIAT_AMOUNT_COLUMN)
+        .should(($div) => {
+          const text = $div.text()
+          expect(text).to.match(fiatRegex)
+        })
+    })
+  })
+
+  describe('fiat currency can be changed', () => {
+    it('should have USD Coin as default currency', () => {
+      // First row Fiat balance should not contain EUR
+      cy.get(balanceRowTestId).first().find('td').eq(FIAT_AMOUNT_COLUMN).should('not.contain', 'EUR')
+      // First row Fiat balance should contain USD
+      cy.get(balanceRowTestId).first().find('td').eq(FIAT_AMOUNT_COLUMN).contains('USD')
+    })
+
+    it('should allow changing the currency to EUR', () => {
+      // Click on balances currency dropdown
+      cy.get('[data-testid=balances-currency-dropdown-btn]').click()
+
+      // Select EUR
+      cy.get('ul[role=menu]').findByText('EUR').click()
+
+      // First row Fiat balance should not contain USD
+      cy.get(balanceRowTestId).first().find('td').eq(FIAT_AMOUNT_COLUMN).should('not.contain', 'USD')
+      // First row Fiat balance should contain EUR
+      cy.get(balanceRowTestId).first().find('td').eq(FIAT_AMOUNT_COLUMN).contains('EUR')
+    })
+  })
+
+  describe('pagination should work', () => {
+    it('should have pagination toolbar', () => {
+      cy.get('.MuiTablePagination-toolbar').contains('Rows per page:')
```

Comment:
I would remove the class and just find the `Rows per page:` text. Although on its own it's pretty useless tbh.